src/services/upload/exifService.ts

```diff
@@ -34,17 +35,31 @@ export async function getExifData(
 }
 
 function getUNIXTime(exifData: any) {
-    const dateString: string = exifData.DateTimeOriginal || exifData.DateTime;
-    if (!dateString || dateString === '0000:00:00 00:00:00') {
+    try {
+        if (!exifData.DateTimeOriginal) {
+            throw Error(CustomError.MISSING_EXIF_TAG);
```

Comment:
If I am understanding it right, this exception will be always logged on sentry.. Is that understanding incorrect?

Reply:
yes it will be

Reply:
Okay. This will probably result in too much noise on sentry and it's acceptable scenario where a photo doesn't have exif data.. Can we only log when something unexpected happens.. like date parsing failed etc etc.